sdk/include/aditof/camera.h

```diff
@@ -463,6 +463,36 @@ class SDK_API Camera {
      */
     virtual Status adsd3500GetGenericTemplate(uint16_t reg,
                                               uint16_t &value) = 0;
+
+  /**
+   * @brief Read serial number from camera and update cache
+   * @return string
+   */
+  virtual aditof::Status readSerialNumber() = 0;
+
+  /**
+   * @brief Set serial number string from ccbContent array
+   * This is internal function
```

Comment:
If this is an internal function, it should not be part of the API. Please move it with the rest of the private methods of the CameraItof.